### ISSUE TITLE
krypton suite install licence

### ISSUE THREAD

**Issue by mkarfaa** [ComponentFactory/Krypton#4]
Dear Sir/Madam
After one month working with your great components, now every time I open my project  it ask for licence key.
Please help me solve this problem.

Best Regards

**Comment by merengue012** [comment]
Hi mkarfaa,
Did you solve that problem? It's asking me for a license key too.

Regards.

**Comment by Cocotteseb** [comment]
Please use the 2013 branch or my fork (https://github.com/Cocotteseb/Krypton)
Once you use the correct source, delete your krypton .licx files in your projects.

**Comment by 21mac** [comment]
Thanks a lot!

I used the fork of Cocotteseb,
then I added the DLLs from "bin/publisherSigned" and the message about licensekey did not show up any longer.

BUT: After I added the controls to the toolbox and dragged one of them to my form,
I get "LC.exe exited with code -1"

Yep, I deleted the "License.licx", which DID contain an entry for the "krypton"-comtrols.
(and I'm not using any repositry).

so I resetted my toolbox, delete any references in my project, but still get the same error and cannot compile my project since then.

Can anyone help please?

**Comment by arpit-nagar** [comment]
I have listed a nuget package at https://www.nuget.org/packages/KryptonWinFormControls/4.4.1
It has batch file KryptonWinFormControls.4.4.1.0\lib\net40\install krypton.bat
Update gacutil path in install.bat 
and run batch file in administrator mode.
Please let me know if you still face the issue.

**Comment by gliwka** [comment]
You can now directly clone the repo, the pull request removing all restrictions has been merged.

**Comment by Tinchio** [comment]
I just downloaded this but when i tried to drag and drop a control to my form a license screen appeared, and it's asking for a serial key. Any ideas?

**Comment by gliwka** [comment]
I've just tested this in a fresh VM and did get no license screen. How did you install the assemblies? Did you take the ones directly in the BIN directory in this repo (not the publisher signed ones)?

**Comment by Tinchio** [comment]
I took the ones in "Bin\PublisherSigned". But it's not that important now, i dismissed that screen and conitinued without license just to try the dlls but i couldn't get them to work. Is there any documentation? i don't know how to open the "Help3.mshc" file. 

    El Sábado, 24 de septiembre, 2016 9:45:51, Matthias Gliwka <notifications@github.com> escribió:
 

 I've just tested this in a fresh VM and did get no license screen. How did you install the assemblies? Did you take the ones directly in the BIN directory in this repo (not the publisher signed ones)?—
